doc/devdocs/log-files.md

```diff
@@ -0,0 +1,5 @@
+# Log files
+
+To read PowerToys log files you can use Analogy Log Viewer with the ddl from @LiorBanai
+
+For more information you can read [#6993](https://github.com/microsoft/PowerToys/issues/6993)
```

Comment:
I would copie the text in this readme instead of referencing the issue. This will also allow us later to easy uodate something of the text.